repl/convert_ast.py

```diff
@@ -7,19 +7,68 @@
 import qwerty_ast
 #################### COMMON CODE FOR BOTH @QPU AND @CLASSICAL DSLs ####################
 
+# quick fix, going to create another file most likely for errors
+class QwertySyntaxError(Exception):
+    pass
+
 def convert_to_qwerty(py_ast):
     print(ast.dump(py_ast, indent=4))
-    value = unpack_ast(py_ast)
-    if value is None:
-        return "Invalid Expression: Not a qubit literal or Tensor"
-    tensor = qwerty_ast.NodeBox.new_qubit_tensor(py_ast.body[0].value.value)
-    return tensor.get_tensor()
 
-def unpack_ast(ast):
+    if not isinstance(py_ast, (ast.Interactive, ast.Module)):
+        return "Invalid AST"
+    
+    if (len(py_ast.body) == 1):
```

Comment:
```suggestion
    if len(py_ast.body) == 1:
```
We don't need the outer parens in Python